src/theme.ts

```diff
@@ -73,7 +229,7 @@ export default (colors: IThemeColors): Theme =>
           borderRadius: 8,
           borderStyle: "solid",
           borderWidth: 1,
-          overflow: "visible"
+          overflow: "hidden"
```

Comment:
Probably this is the cause of input issue - without this change, it displayed popup also outside of the card:
<img width="347" alt="Zrzut ekranu 2020-12-2 o 13 23 02" src="https://user-images.githubusercontent.com/9825562/100872469-267cee00-34a2-11eb-906c-6395ba2d9159.png">

Reply:
Aw damn this is part of debug code 🙄